snooty/rstspec.toml

```diff
@@ -1,3 +1,5 @@
+data_fields = ["source_page_template", "assets"]
```

Comment:
What's `source_page_template` ?

Reply:
This may change, but I wanted to start getting infrastructure in for communicating source URLs to the frontend :D